apps/election-manager/src/utils/CastVoteRecordFiles.ts

```diff
@@ -278,6 +336,31 @@ export default class CastVoteRecordFiles {
   public get castVoteRecords(): CastVoteRecordLists {
     return this.allCastVoteRecords
   }
+
+  /**
+   * Gets the file mode for the set of CVR files.
+   */
+  public get fileMode(): CastVoteRecordFileMode | undefined {
+    let liveSeen = false
+    for (const cvrs of this.allCastVoteRecords) {
+      for (const cvr of cvrs) {
+        if (cvr._testBallot) {
+          return 'test'
+        }
+        liveSeen = true
+      }
+    }
+    return liveSeen ? 'live' : undefined
+  }
+
+  public filenameAlreadyImported(filename: string): boolean {
+    for (const f of this.files) {
+      if (f.name === filename) {
```

Comment:
```suggestion
    for (const file of this.files) {
      if (file.name === filename) {
```

Or, if you don't want to name the variable:

```suggestion
    for (const { name } of this.files) {
      if (name === filename) {
```